lib/Backend/Lower.cpp

```diff
@@ -3023,6 +3023,7 @@ Lowerer::LowerRange(IR::Instr *instrStart, IR::Instr *instrEnd, bool defaultDoFa
             instr->Remove();
             break;
 
+#endif //ENABLE_WASM
         case Js::OpCode::SpeculatedLoadFence:
         {
             AssertOrFailFast(instr->m_kind == IR::InstrKindByteCodeUses);
```

Comment:
Looks like the next line declaring `bcuInstr` needs to be part of the `#else` since arm is complaining it is declared but not used.